nuxeo-runtime/nuxeo-stream/kafka-docker/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '2'
+services:
+  zookeeper:
+    image: wurstmeister/zookeeper:latest
```

Comment:
Is `wurstmeister` an "official" repository? If not we should look for one...

Reply:
ok I will take the zookeeper official image